src/main/java/uk/gov/hmcts/darts/common/entity/RetentionPolicyTypeEntity.java

```diff
@@ -42,4 +43,9 @@ public class RetentionPolicyTypeEntity extends CreatedModifiedBaseEntity {
     @Column(name = "policy_end_ts")
     private OffsetDateTime policyEnd;
 
+    static Policy getEnumerationForPolicyName(String policyName) {
```

Comment:
`Policy.getForPolicyName(policyName)`  could just be called directly, maybe I'm missing something but not sure theres value in wrapping in the entity class as it doesn't use any of the entity's data.

Reply:
Agreed, now removed